unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
This isn't really a header file (in fact, I don't like that the extension is .h). It is being parsed programatically by py/makeqstrdata.py to generate a proper header file so no ifdef's are possible at this stage. The other header file that references the generated header file is also being created programatically, so it's a catch-22